etc/containerpilot.json

```diff
@@ -17,6 +17,14 @@
       "onChange": "python /usr/local/bin/manage.py on_change"
     }
   ],
+  "tasks": [
+    {
+      "name": "snapshot",
+      "command": "python /usr/local/bin/manage.py snapshot_task",
+      "frequency": 5,
```

Comment:
I was also concerned that a 5 second interval was awfully fast, but upon reading the code, we're only checking to see if we _should_ back up every 5 seconds. Actual backup frequency is defined elsewhere.

Is it worth renaming this `snapshot_check` to clarify that?